scripts/output

```diff
@@ -141,3 +143,74 @@ pip:
 rep: "$REP"
 EOM
 }
+
+function copy_template
+{
+  if [ -z "$1" ] || [ -z "$2" ]; then
+    echo "No input or output file passed. Use copy_template path/to/input.template path/to/output.json"
+    exit 1
+  fi
+
+  if [ -f "$2" ]; then
+    echo "Output configuration file already exist. Delete it [y/n] ?"
+    read yn
+    if [[ $yn =~ ^[Yy]$ ]]; then
+      rm "$2"
+    else
+      echo "Nothing else to do! Exiting..."
+      exit 1
+    fi
+  fi
+  cp $1 $2
+}
+
+function write_template
+{
+  if [ -z "$1" ]; then
+    echo "No output file passed. Use write_template path/to/file.json"
+    exit 1
+  fi
+
+  echo "Publishing output into $1"
+  
+  sed -i -e "s/{{TESTCHAIN_HOST}}/$HOST/g" $1
```

Comment:
Would something like `sed -i "s/{{TESTCHAIN_\(.*\)}}/$\1/g" $1` work as a way not to have to repeat all these `sed`s and just use one.

Also you don't need the `-e`, as you're only passing one pattern for each invocation.

Reply:
@asymmetric makes sense, but I did it to be sure what variables are there and what missing. We will change deploy section a bit later and refactor this part

Reply:
Without `-e` it wouldn't work correctly on osx.
```
➜  ~  sed -i "s/test/temp/g" ./test
sed: 1: "./test": invalid command code .
➜  ~  sed -i -e "s/test/temp/g" ./test
➜  ~
```

Reply:
Ah right, macOS ships with BSD `sed`.

Reply:
I've also used this workaround for macos `sed`:
```shell

function sed_inplace {
  # sed's -i argument behaves differently on macOS, hence this hack
  sed -i.bak "$1" $2 && rm $2.bak
}
```
so then the line would be:
```suggestion
sed_inplace "s/{{TESTCHAIN_HOST}}/$HOST/g" $1
```

Reply:
Or mention that the script requires GNU sed ;)

Reply:
Right now it works perfectly on OSX and Linux. I removing file that `sed` created after all replacements. 
I don't see any reason to remove it after each step. 
And installing GNU sed to make it work without `-e` seems a little bit weird.

Reply:
I do not think we should require mac users to install GNU sed to use this script.